backend/openapi.yaml

```diff
@@ -926,6 +930,19 @@ paths:
           schema:
             type: string
             example: b477389cb3514fb1b444052a39bfb65d
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              description: TBD
+              type: object
+              additionalProperties: false
+              properties:
+                additional_info:
+                  description: Additional information to be displayed on the generated PDF
```

Comment:
`additional_info` is the correct translation but I'm thinking is a `notes` a better term (we already use that). `additional_info` sounds more like an object than a string. Other option would be to add "Additional **textual** information" to the description